requirements/requirements.txt

```diff
@@ -10,5 +10,6 @@ torchvision
 wget
 wrapt
 ruamel.yaml
-sklearn
+scikit-learn
```

Comment:
My problem with that is that I do not think that NeMo Core should rely on scikit-learn.

I see references in: NLP callbacks and some ASR related scripts.

Shouldn't that go to requirements-asr in that case?

@okuchaiev Where is the "boarder" of requirements.txt? 

I am approving that PR but I do not understand that.

Reply:
Wasn't it already in requirements?

That's a good question. I feel like a proper answer is only the libraries that are required in files not under `collections` should be in this file.
But the more practical answer seems to be if it is inside asr and nlp, then it might be worth elevating to be under requirements.